### DISCUSSION TITLE
how to set Default Value with jalali date ?

### DISCUSSION THREAD

**Discussion by alirezaloveimi** [gpbl/react-day-picker#2057] [Translation and localization]
how to set Default Value with jalali date ? 
i from iran and we use the jalali date

**Comment by gpbl** [comment]
Hi @alirezaloveimi,

we are approaching the release of v9 which will include support for Jalali:
https://react-day-picker.js.org/next/using-daypicker/localization#jalali-calendar

We'd appreciate you giving a chance to test it and share your experience with it - thanks!

**Reply by behnam-hs** [reply]
i did test it and it's not working yet !

**Reply by gpbl** [reply]
@behnam-hs what do you mean?

**Comment by behnam-hs** [comment]
Dude differences between Jalali date-format and gregorian or hijrj dates is
huge. Its smt more than translating the months name or add a number to
time.
It didnt work means dates are not accurate in the jalali locale. compare
with time.ir!

**Reply by gpbl** [reply]
@behnam-hs please, some constructive feedback. thanks.
